### PR TITLE
fix asyncio task depreciation

### PR DESCRIPTION
**Pull request by idowu-adeleke**
## Summary:
Describe what your change is supposed to do.
 Update Peewee-asyn patch to support python 3.9

## JIRA Issue:
https://peopleai.atlassian.net/browse/MATCH-48
https://peopleai.atlassian.net/browse/MATCH-350

## Test Plan:
Describe how you tested the change and how someone else can repeat the test.

## Checklist

- [x] Bumped version
- [ ] The PR is assigned on reviewer(s)
- [x] JIRA Issue is linked into the PR

### CODE REVIEW

File: peewee_async.py [hunk 1]
```diff
@@ -161,49 +168,47 @@ async def my_async_func():
             model = source_
 
         conditions = list(args) + [(getattr(model, k) == v)
-            for k, v in kwargs.items()]
+                                   for k, v in kwargs.items()]
```
Comment (ChinmaySKulkarni) [inline peewee_async.py]: nit: remove whitespace change

File: peewee_async.py [hunk 2]
```diff
@@ -888,60 +869,57 @@ def connect_async(self, loop=None, timeout=None):
...
         else:
             conn = None
         try:
-            return (yield from self._async_conn.cursor(conn=conn))
-        except asyncio.CancelledError:
-            raise
-        except Exception as exc:
-            yield from self.close_async()
+            return (await self._async_conn.cursor(conn=conn))
+        # except asyncio.CancelledError:
```
Comment (ChinmaySKulkarni) [inline peewee_async.py]: Are you sure this change is required? If yes, remove commented code

**Review by ChinmaySKulkarni — APPROVED** [review]
Looks good! Some minor comments. There's some API changes which I'm wary of, can you mention what all was done to test this?

File: peewee_async.py (hunk 1 above)
Comment (idowu-adeleke) [reply peewee_async.py]: fixed

File: peewee_async.py (hunk 2 above)
Comment (idowu-adeleke) [reply peewee_async.py]: removed

**Pull request merged**